geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/LogsAreFullyRedactedAcceptanceTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.commands;
+
+import static org.apache.geode.distributed.ConfigurationProperties.LOG_LEVEL;
+import static org.apache.geode.distributed.ConfigurationProperties.SECURITY_MANAGER;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Properties;
+import java.util.Scanner;
+
+import org.apache.commons.io.FileUtils;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.examples.SimpleSecurityManager;
+import org.apache.geode.management.internal.cli.util.CommandStringBuilder;
+import org.apache.geode.test.junit.categories.AcceptanceTest;
+import org.apache.geode.test.junit.rules.gfsh.GfshRule;
+
+@Category(AcceptanceTest.class)
+public class LogsAreFullyRedactedAcceptanceTest {
+
+  private static String PASSWORD = "abcdefghijklmn";
+
+  // Don't put "password" in all the key names, since that is one of our taboo words.
+  private static final String gfshSysEnvUsername = "log-no-test.security-username";
+  private static final String gfshSysEnvUsernameValue = "test.sysenv-username";
+  private static final String gfshSysEnvPassword = "log-no-test.security-password";
+  private static final String gfshSysEnvPasswordValue = PASSWORD + "-sysenv";
+
+  private static final String gfshSysEnvUsernamePost = "security-username.log-no-test";
+  private static final String gfshSysEnvUsernameValuePost = "sysenv-username.test";
+  private static final String gfshSysEnvPasswordPost = "security-password.log-no-test";
+  private static final String gfshSysEnvPasswordValuePost = PASSWORD + "-post";
+
+
+
+  private File propertyFile;
+  private File securityPropertyFile;
+
+  @Rule
+  public GfshRule gfsh = new GfshRule();
+
+  @BeforeClass
+  public static void addSecuritySystemProperties() {
+    System.setProperty(gfshSysEnvUsername, gfshSysEnvUsernameValue);
```

Comment:
what would these System.setProperty calls do? you are only setting the sys properties in the junit VM. If you want to have these set in the server/locator VMs, you need to pass them in --J options when you start the server/locator.

Reply:
I had thought that members pick up the `gfsh` environment, and that this would percolate to that.  I appear to have been wrong, though.